facia-press/app/controllers/HealthCheck.scala

```diff
@@ -11,13 +12,17 @@ class HealthCheck(
   toolPressQueueWorker: ToolPressQueueWorker,
   val controllerComponents: ControllerComponents
 )
-  extends HealthCheckController with Results {
+  extends HealthCheckController with Results with Logging {
   val ConsecutiveProcessingErrorsThreshold = 5
   override def healthCheck(): Action[AnyContent] = Action.async{
     if (!toolPressQueueWorker.lastReceipt.exists(_.plusMinutes(1).isAfter(DateTime.now))) {
-      successful(Results.InternalServerError("Have not been able to retrieve a message from the tool queue for at least a minute"))
+      val msg = "Have not been able to retrieve a message from the tool queue for at least a minute"
```

Comment:
I know you didn't introduce this but what are your thoughts on extracting out the `1` from `_.plusMinutes(1)` on the line above here to a variable and interpolating it here into the log message? That would tie the two lines together and make impossible to update one value without updating the other.